website/src/pages/index.tsx

```diff
@@ -22,59 +22,59 @@ type FeatureItem = {
 
 const FeatureList: FeatureItem[] = [
   {
-    title: "Pub/Sub logging",
+    title: "Pub/sub logging",
     Icon: icons.Robot,
     description:
-      "MCAP is ideal for storing multiple channels of timestamped log data, such as pub/sub messages or multimodal sensor data.",
+      "Store multiple channels of timestamped log data, including pub/sub messages or multimodal sensor data.",
   },
 
   {
-    title: "Serialization agnostic",
+    title: "Serialization-agnostic format",
     Icon: icons.DrawerEnvelope,
     description:
-      "MCAP is a container format, allowing you to record and replay binary messages in any format, such as Protobuf, DDS (CDR), ROS, JSON, etc.",
+      "Record and replay binary messages in any format – like Protobuf, DDS (CDR), ROS, JSON, and more.",
   },
   {
     title: "High-performance writing",
     Icon: icons.SportsCarConvertible,
     description:
-      "MCAP utilizes a row-oriented, append-only design. This minimizes disk I/O, and reduces the risk of data corruption during an unclean shutdown. ",
+      "Use MCAP's row-oriented, append-only design to minimize disk I/O and reduce the risk of data corruption during unclean shutdowns.",
```

Comment:
This has changed the meaning, MCAP's row-oriented design is not something I "use" as a feature. It is part of the MCAP design.

```suggestion
      "MCAP uses a row-oriented, append-only design to minimize disk I/O and reduce the risk of data corruption during unclean shutdowns.",
```